mc2/controllers/base/models.py

```diff
@@ -54,6 +54,8 @@ class Controller(PolymorphicModel):
             'is automatically set each time the item is saved.')
     )
 
+    webhook_token = models.UUIDField(null=True)
```

Comment:
We probably could, but I have a mild preference for not exposing the webhook unless it's going to be used.